content/riak/kv/3.2.0/configuring/next-gen-replication.md

```diff
@@ -53,14 +111,13 @@ Setting | Options | Default | Description
 `ttaaefs_daycheck` | `any` (integer) | `0` | How many times per 24hour period should the last 24-hours of data be checked to confirm it is fully sync'd.
 `ttaaefs_rangecheck` | `any` (integer) | `0` | How many times per 24hour period should the a range_check be run.
 `ttaaefs_logrepairs` | `enabled`, `disabled` | `enabled` | If Tictac AAE full-sync discovers keys to be repaired, should each key that is repaired be logged
-`tictacaae_active` | `active`, `passive` | `passive` | Enable or disable tictacaae. Note that disabling tictacaae will set the use of tictacaae_active only at startup - setting the environment variable at runtime will have no impact.
+`tictacaae_active` | `active`, `passive` | `passive` | Enable or disable TicTac AAE. Note that disabling TicTac AAE will set the use of tictacaae_active only at startup - setting the environment variable at runtime will have no impact.
 `aae_tokenbucket` | `enabled`, `disabled` | `enabled` | To protect against unbounded queues developing and subsequent timeouts/crashes of the AAE process, back-pressure signalling is used to block the vnode should a backlog develop on the AAE process. This can be disabled.
 `tictacaae_dataroot` | `` | `"$platform_data_dir/tictac_aae"` | Set the path for storing tree caches and parallel key stores. Note that at startup folders may be created for every partition, and not removed when that partition hands off (although the contents should be cleared).
-`tictacaae_parallelstore` | `leveled_ko`, `leveled_so` | `leveled_so` | On startup, if tictacaae is enabled, then the vnode will detect of the vnode backend has the capability to be a "native" store. If not, then parallel mode will be entered, and a parallel AAE keystore will be started. There are two potential parallel store backends - leveled_ko (Key ordered leveled), and leveled_so(Segment ordered Leveled).
+`tictacaae_parallelstore` | `leveled_ko`, `leveled_so` | `leveled_so` | On startup, if TicTac AAE is enabled, then the vnode will detect of the vnode backend has the capability to be a "native" store. If not, then parallel mode will be entered, and a parallel AAE keystore will be started. There are two potential parallel store backends - leveled_ko, and leveled_so.
```

Comment:
Why remove the explanation of so and ko?

Reply:
Ah this is something that was removed from 3.0.2 to 3.0.3 for some reason.. I'll readd it.